### DISCUSSION TITLE
Nano Tesla as Dimensional Contraint

### DISCUSSION THREAD

**Discussion by AudoneGianluca91** [MilesCranmer/PySR#733] [Q&A]
I have a simple question how can I have nano Tesla as dimensional contraint? I have tried with nT but it say the unit is not recognised.

Thank you very much for your help!

**Comment by MilesCranmer** [comment]
For dimensionality constraints, the specific unit does not matter, only dimensions. Basically the units themselves are ignored. (I considered both options but there seemed to be too many gotchas if we actually took into account unit values)

However we could definitely add nT to the units library, which is developed here: https://github.com/SymbolicML/DynamicQuantities.jl. My policy is to basically add a unit whenever someone requests it (rather than adding a bunch of units that nobody uses)

**Reply by MilesCranmer** [reply]
And the unit (and prefix) list is here: https://github.com/SymbolicML/DynamicQuantities.jl/blob/main/src/units.jl
